regression-test/data/data_model/unique/test_unique_table_like.out

```diff
@@ -5,11 +5,13 @@ int_value	INT	Yes	false	\N	REPLACE
 char_value	CHAR(10)	Yes	false	\N	REPLACE
 date_value	DATE	Yes	false	\N	REPLACE
 __DORIS_DELETE_SIGN__	TINYINT	No	false	0	REPLACE
+__DORIS_SEQUENCE_COL__	INT	Yes	false	\N	REPLACE
```

Comment:
I don't see the code add `__DORIS_SEQUENCE_COL__`, can you help me point it out

Reply:
Before fix this bug, when create table b like table a which having column `__DORIS_SEQUENCE_COL__`, i can not get table b with column `__DORIS_SEQUENCE_COL__`. 

it is diffrent between able a and table b.

when get schema of table a, return table's `function_column.sequence_type` property, and you will get table b with column `__DORIS_SEQUENCE_COL__ ` like table a.